services/ui-src/src/forms/mcpar/cpc/cpc.json

```
@@ -11,8 +15,129 @@
+              "name": "cpc-3-o2",
+              "type": "choice",
+              "label": "Prepaid Inpatient Health Plan (PIHP)"
+            },
+            {
+              "name": "cpc-3-o3",
+              "type": "choice",
+              "label": "Prepaid Ambulatory Health Plan (PAHP)"
+            },
+            {
+              "name": "cpc-3-o4",
+              "type": "choice",
+              "label": "Primary Care Case Management (PCCM) Entity"
+            },
+            {
+              "name": "cpc-3-o5",
+              "type": "choice",
+              "label": "Other, specify",
+              "children": [
+                {
+                  "id": "cpc-3-o5-text",
+                  "type": "textarea"
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        "id": "cpc-4",
+        "type": "checkbox",
+        "props": {
+          "label": "C1.I.4a Special program benefits",
+          "hint": "Are any of the four special benefit types covered by the managed care program: (1) behavioral health, (2) long-term services and supports, (3) dental, and (4) transportation, or (5) none of the above? Select one or more.</br>Only list the benefit type if it is a covered service as specified in a contract between the state and managed care plans participating in the program. Benefits available to eligible program enrollees via fee-for-service should not be listed here.",
+          "choices": [
+            {
+              "name": "cpc-4-o1",
+              "type": "choice",
+              "label": "Behavioral health"
+            },
+            {
+              "name": "cpc-4-o2",
+              "type": "choice",
+              "label": "Long-term services and supports"
+            },
+            {
+              "name": "cpc-4-o3",
+              "type": "choice",
+              "label": "Dental"
+            },
+            {
+              "name": "cpc-4-o4",
+              "type": "choice",
+              "label": "Transportation"
+            },
+            {
+              "name": "cpc-4-o5",
+              "type": "choice",
+              "label": "None of the above",
+              "children": [
+                {
+                  "id": "cpc-4-o5-text",
+                  "type": "textarea"
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        "id": "cpc-5",
+        "type": "textarea",
+        "props": {
+          "label": "C1.I.4b Variation in special benefits",
+          "hint": "What are any variations in the availability of special benefits within the program (e.g. by service area or population)?"
+        }
+      },
+      {
+        "id": "cpc-6",
+        "type": "number",
+        "props": {
+          "label": "C1.I.5 Program enrollment",
+          "hint": "Enter the total number of individuals enrolled in the managed care program as of the first day of the last month of the reporting year."
+        }
```

To get the number field to mask correctly with thousands separators, add the following to props:
```suggestion
        "props": {
          "label": "C1.I.5 Program enrollment",
          "hint": "Enter the total number of individuals enrolled in the managed care program as of the first day of the last month of the reporting year.",
          "mask": "comma-separated"
        }
```